packages/react/src/jsx.tsx

```diff
@@ -12,14 +12,20 @@ import {
 } from './rise'
 
 type ServerComponent = ComponentModelState<ServerEventModelState>
-
-export const jsxs = jsx
-
-export function jsx(
+type JSXFactory = (
   componentFactory: ((props: any) => ServerComponent | ReactElement) | undefined,
   { children, ...passedProps }: Record<string, any>,
   key?: string
-): ServerComponent {
+) => ServerComponent
+
+export const jsxs: JSXFactory = (...args) => {
```

Comment:
We generally prefer function declaration over arrow function as a top-level export, but this is meant to be used internally by React, so it's a fair tradeoff. The reason was to share `JSXFactory` type between these two w/o repeating.